### DISCUSSION TITLE
How to subscribe ?

### DISCUSSION THREAD

**Discussion by BornTKill** [Status-Page/Status-Page#464] [Q&A] [answered]
Is subscription for notification only available thanks to API ?
I am a little bit confused.

**Comment by BornTKill** [comment]
I found settings in statuspage/config/parameters.py
That's the good way to proceed ?

**Comment by herrtxbias** [comment, marked as answer]
If you want to enable the Subscription Button you need to add a "Configuration Revision" in the Admin Panel and enable "Subscriptions" there. Please note, that you need to configure a SMTP Server in your `statuspage/configuration.py` before. The File `statuspage/config/parameters.py` however is not a configuration file.

**Comment by BornTKill** [comment]
That's works like this. Better indeed :)
